web/html/src/components/input/Form.js

```diff
@@ -76,14 +76,18 @@ export class Form extends React.Component<Props> {
   }
 
   unregisterInput(component: React.ElementRef<any>) {
-    if (component.props && component.props.name && this.inputs[component.props.name] === component) {
-      delete this.inputs[component.props.name];
+    if (component.props && component.props.name){
+      const name = component.props.name instanceof Array ? component.props.name.join() : component.props.name;
+      if (this.inputs[name] === component) {
+        delete this.inputs[component.props.name];
```

Comment:
This looks like a bug, or perhaps I'm misunderstanding the intent here? I think this should be `delete this.inputs[name]`, otherwise we're passing the array as the object key (if we're in a setup where it is an array).

Reply:
oh right! thanks for catching it!

Reply:
done